docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,163 @@
+******************************
+Adding or Removing a PBFT Node
+******************************
+
+Membership of a Sawtooth PBFT network is controlled by the on-chain setting
+``sawtooth.consensus.pbft.members``. An administrator must update this setting
+when :ref:`adding a new node <adding-a-pbft-node-label>` or
+:ref:`removing an existing node <removing-a-pbft-node-label>`.
+
+All nodes in the network monitor this setting. When a node detects a change,
+it updates its local list of PBFT members to match the new list. When all member
+nodes have updated the setting, consensus resumes normal operation.
+
+.. _adding-a-pbft-node-label:
+
+Adding a PBFT Node
+==================
+
+To add a new node to an existing PBFT network, you will install and configure
+the node, start it and wait for it to catch up with the rest of the network,
+then update ``sawtooth.consensus.pbft.members`` on an existing node.
+
+You can add several nodes at the same time.
+
+1. Install and configure Sawtooth on the new node, as described in
+   :doc:`setting_up_sawtooth_poet-sim`.
+   Skip the procedure to create a genesis block.
+
+#. Start the new node. It will join the Sawtooth network and start the block
+   catch-up procedure, where it receives and commit the blocks that are already
+   on the blockchain. (For details, see the `PBFT node catch-up RFC
+   <https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0031-pbft-node-catchup.md>`_.)
+
+   The new node sends consensus messages as processes blocks, but the other
```

Comment:
Not sure this sentence is necessary; it's true, I don't think it's necessary. I'm open to other opinions though.

Reply:
I removed the sentence.